### ISSUE TITLE
Flush output stream after each call/return

### ISSUE THREAD

**Issue by weissjeffm** [RedHatQE/function_trace#8]
```
12:13 <mfalesni> jweiss-lunch: can you add .flush() to the function trace plugin?
12:13 <mfalesni> jweiss-lunch: it is better during live trace
```

@mfalesni this should be straighforward, will take a stab at it.  I think that might solve the issue in ipython where tracebacks appear in the middle of the trace output.  i had assumed the issue was with either ipython (or the emacs client end) but i'll try this and see if it helps.

**Comment by mfalesni** [comment]
I also think that the function tracing plugin in cfme_tests could also write its own log file where it would log everything, so you could `tail -f` it during tests. But that is just an idea :)

**Comment by weissjeffm** [comment]
Fixed in df27326
